src/Containers/Pages/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './LandingPage.css';
+import Helmet from 'react-helmet';
+
+const LandingPage = () => {
+    return (
+        <>
+            <Helmet bodyAttributes={{style: 'background-color : #FFD275'}}/>
+            <h3 className='logo'> PlatePal</h3>
+            <button className='login-button'>Login</button>
```

Comment:
instead of referring to CSS classes directly, try importing it as a module. This benefits later on where if there are overlaps in class names, the styles from different files extrude and affect other components. Try to see the reference component I created in MetricBtn.js

Reply:
you will see **_className={styles.text}_** or **_className={styles.btnSubLabel}_** like this